### ISSUE TITLE
brokenmass-RailgunsRetargeting-1.3.3 orbit 30 wasting sails

### ISSUE THREAD

**Issue by raziel29a** [DysonSphereMod/QOL#126]
When no possible orbit is possible but aiming directly at the star is: orbit (30) is targeted which isn't really an orbit. Sails launched at orbit 30 hit the star directly and despawn. It's not possible to have them absorbed by the dyson sphere, they produce no power.

**Comment by raziel29a** [comment]
Upon further investigation the issue is only visual. Sails disappear into the star but still are being consumed by the Dyson Sphere and provide energy production, however are no longer visible.

**Comment by EstebanLB** [comment]
What happens if there is no D Sphere, only Swarm? Are they destroyed?

**Comment by raziel29a** [comment]
Nope, just not visible. I've manipulated the camera to see if they end up inside the star and they simply vanish, although their power output is still there. I guess their stats aren't hard coded with the visual object and the orbit (30) issue affects them only on rendering level.
